### DISCUSSION TITLE
Ability to reset search when search box has null value

### DISCUSSION THREAD

**Discussion by rmillener** [microsoft-search/pnp-modern-search#2023] [Ideas]
The PnP search box hides the cancel/reset button when it has a null value. This creates issues when a user wants to get back to the default query but have already blanked the search field. 

As an example, in this case I have searched for "Dog" then blanked the search field. I want to reset the search back to its default all items ( * ) query but don't see the "x" that would allow me to do so.
![image](https://user-images.githubusercontent.com/36008992/163279687-aae0c1df-bae0-46ad-ba37-197a6f092854.png)

I'm interested in any feedback about whether the reset button should always show. I'm happy to submit a PR if this is desirable. As an alternative we are likely to implement a breadcrumb that gives the user a "home" link to reset the query.

**Comment by wobba** [comment]
This is by design, removing text in the search box means execute a new query from the user - not an automatic re-query. I understand what you want to accomplish, but this behavior is not intended. Same goes for any OOB search page you set up with classic/modern. Clearing the search box will not re-fire a query with the default result template and `*`.

**Reply by rmillener** [reply]
thanks for the response, we will go with the breadcrumb
